dvl-a50/static/js/rotatedmarker.js

```diff
@@ -0,0 +1,57 @@
+(function() {
+  // save these original methods before they are overwritten
+  var proto_initIcon = L.Marker.prototype._initIcon;
+  var proto_setPos = L.Marker.prototype._setPos;
+
+  var oldIE = (L.DomUtil.TRANSFORM === 'msTransform');
+
+  L.Marker.addInitHook(function () {
+      var iconOptions = this.options.icon && this.options.icon.options;
+      var iconAnchor = iconOptions && this.options.icon.options.iconAnchor;
+      if (iconAnchor) {
+          iconAnchor = (iconAnchor[0] + 'px ' + iconAnchor[1] + 'px');
+      }
+      this.options.rotationOrigin = this.options.rotationOrigin || iconAnchor || 'center bottom' ;
+      this.options.rotationAngle = this.options.rotationAngle || 0;
+
+      // Ensure marker keeps rotated during dragging
```

Comment:
```suggestion
      // Ensure marker keeps rotating during dragging
```

Reply:
this is external code, just included here to be available offline, I'll avoid touching it